barretenberg/cpp/src/barretenberg/eccvm/eccvm_prover.cpp

```diff
@@ -47,18 +47,32 @@ void ECCVMProver::execute_preamble_round()
  */
 void ECCVMProver::execute_wire_commitments_round()
 {
+    // We mask the last MASKING_OFFSET coefficients of each wire polynomial. To commit to the masked wires when
```

Comment:
Maybe: We add MASKING_OFFSET random values to the coefficients of each wire polynomial to not leak information via the commitment and evaluation.

.. or something along these lines